sources/packages/web/src/components/aest/students/modals/ApproveApplicationOfferingChangeRequestModal.vue

```diff
@@ -0,0 +1,109 @@
+            appendix;
+          </v-list-item>
+          <v-list-item>
+            2. The student's school will have access to information provided in
+            this appendix;
+          </v-list-item>
+          <v-list-item>
+            3. The information in this appendix is subject to audit,
+            investigation and verification as defined in the current program
+            year's StudentAid BC Policy Manual
+          </v-list-item>
+        </v-list>
+        <v-row
+          ><v-col
+            ><h4 class="label-bold">Collection and use of information</h4>
+          </v-col></v-row
+        >
+        <v-row
+          ><v-col
+            >The information included in this form and authorized above is
+            collected and managed in accordance with sections 26(c) and 26(e) of
+            the Freedom of Information and Protection of Privacy Act, R.S.B.C.
+            1996, c. 165, and under the authority of the Canada Student
+            Financial Assistance Act, R.S.C. 1994, Chapter C-28 and StudentAid
+            BC. The information provided will be used to determine eligibility
+            for a benefit through</v-col
+          ></v-row
+        > </content-group-info
+      ><v-checkbox
+        ref="declarationCheckbox"
+        v-model="approveApplicationOfferingChangeRequestModal.studentConsent"
+        label="I agree to the terms and conditions of the StudentAid BC Declaration
+          form"
+      ></v-checkbox>
+    </template>
+    <template #footer>
+      <footer-buttons
+        primaryLabel="Allow change"
+        @secondaryClick="cancel"
+        @primaryClick="allowChange"
+      />
+    </template>
+  </modal-dialog-base>
+</template>
+<script lang="ts">
+import ModalDialogBase from "@/components/generic/ModalDialogBase.vue";
+import { ref, defineComponent, watch } from "vue";
+import { useModalDialog } from "@/composables";
+import { UpdateApplicationOfferingChangeRequestAPIInDTO } from "@/services/http/dto";
+import { ApplicationOfferingChangeRequestStatus } from "@/types";
+
+export default defineComponent({
+  components: {
+    ModalDialogBase,
+  },
+  setup() {
+    const { showDialog, showModal, resolvePromise } = useModalDialog<
+      UpdateApplicationOfferingChangeRequestAPIInDTO | boolean
+    >();
+    const approveApplicationOfferingChangeRequestModal = ref(
+      {} as UpdateApplicationOfferingChangeRequestAPIInDTO,
+    );
+    watch(showDialog, async (newshowDialogValue) => {
+      if (newshowDialogValue) {
+        approveApplicationOfferingChangeRequestModal.value.studentConsent =
+          false;
+      }
+    });
+    const cancel = () => {
+      resolvePromise(false);
+    };
+    const allowChange = async () => {
+      approveApplicationOfferingChangeRequestModal.value.applicationOfferingChangeRequestStatus =
+        ApplicationOfferingChangeRequestStatus.InProgressWithSABC;
+      resolvePromise(approveApplicationOfferingChangeRequestModal.value);
```

Comment:
did u test the validation?  and shouldn't we need `value.validate();` and `reset` here?